products/health-care/beneficiary-travel/engineering/data-flow-sequence-diagram.md

```diff
@@ -0,0 +1,33 @@
+The basic sequence for most BTSSS is straightforward:
```

Comment:
Nice high-level diagram! I'm wondering to what level we should address these points on the Readiness Review ticket:

* Data Flow Diagram:
  * This diagram must illustrate the following specifics.
  * What data is collected or used, and where, including information such as credentials used by this system?
  * Where is the data is stored and how, including information such as any encryption used?
  * How is the data transferred, including information such as any encryption used?
  * Who accesses the data and in what capacity (read or read-write)?
  * What is the audit trail of data access and manipulation?

Reply:
I'm leaning toward answering these in-diagram from our (VA.gov) perspective and not try to answer for the backing API team.

What do you think?

Reply:
I think that's fine, whether that's in the diagram or any additional information below the diagram

Reply:
Let's add it in the diagram. Let me know if you need help fitting something into diagram form